### PR TITLE
Initiating design, adding:

### PR DESCRIPTION
**Pull request by TheMarv**
- Global Styles
- Themes
- Layout

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,20 +15,22 @@
   "dependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
+    "@mui/icons-material": "^5.8.0",
     "@mui/material": "^5.8.1",
     "@next-auth/mongodb-adapter": "^1.0.3",
     "mongodb": "^4.6.0",
     "mongoose": "^6.3.3",
+    "nanoid": "^3.3.4",
```
Comment (HishamAlmannai) [inline package.json]: Not in use. Should be installed later.

File: pages/index.jsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { Grid } from '@mui/material';
+import Link from 'next/link';
```
Comment (HishamAlmannai) [inline pages/index.jsx]: Both not in use. Can be removed.

**Review by HishamAlmannai — APPROVED** [review]
This is a very different working progress than I have learned here. So I cannot say a lot about it. You can discuss the following with your course:
If you would  work by MVP (Minimal Viable Product) you would probably not start with styling. 
I did not find a UserStory / Issue concerning this pull request. So I cannot say if the pull request is fulfilled.
Our couches did not recommend mui for our capstone projects.

**Review by marcbendixen — APPROVED** [review]
LGTM 🚀

**Pull request merged**